agent/engine/docker_task_engine.go

```diff
@@ -597,6 +609,19 @@ func (engine *DockerTaskEngine) createContainer(task *api.Task, container *api.C
 	seelog.Infof("Created container name mapping for task %s - %s -> %s", task, container, containerName)
 	engine.saver.ForceSave()
 
+	//Initialize metadata file 
+	//TODO: Do initial write of static data to the file
+	metadataPath, ioerr := metadataservice.InitMetadataFile(task, container)
+	if ioerr == nil {
+		seelog.Infof("Created metadata file at %s", metadataPath)
+	} else {
+		seelog.Errorf("Failed to create metadata file at %s. Error: %s", metadataPath, ioerr.Error())
+	}
+	//Bind host volume to mount path in container
+	hostMetadataPath := HostDataDir + metadataservice.GetMetadataFilePath(task, container)
+	hostConfig.Binds = []string{ hostMetadataPath + ":" + "/ecs/metadata/" + container.Name }
```

Comment:
can you make `/ecs/metadata` a constant, or format of the binds, eg: `"%s/ecs/metadata/%s"`. And also extract this part as a function?